### PR TITLE
Release note for v0.10.0

### PR DESCRIPTION
**Pull request by andelf**
- [[Features]]
	- A new and improved [[Search]]! [#10132](https://github.com/logseq/logseq/pull/10132)
	- Design core enhancements including the ability to set an "Accent color" from Settings [#10132](https://github.com/logseq/logseq/pull/10132)
- [[Thanks]]
	- [[cosmo9able]]
		- Update Brazilian Portuguese translation in [#10461](https://github.com/logseq/logseq/pull/10461)
	- [[e-zz]]
		- Fix wrong bounding coordinates for PDF highlights in [#10426](https://github.com/logseq/logseq/pull/10426)
	- [[eliovir]]
		- Update French translation in [#10454](https://github.com/logseq/logseq/pull/10454)
	- [[ormai]]
		- Update Italian translation in [#10557](https://github.com/logseq/logseq/pull/10557)
	- [[queeup]]
		- Update Turkish translation in [#10492](https://github.com/logseq/logseq/pull/10492)
- [[Fixed issues]]
	- Remove the need to add a space to trigger search command [#10506](https://github.com/logseq/logseq/pull/10506)
	- Fix background color for some UI buttons [#10512](https://github.com/logseq/logseq/pull/10512)
	- Fix demo page creation for browser version [#10497](https://github.com/logseq/logseq/pull/10497)
	- Fix wrong bounding coordinates for PDF highlights [#10426](https://github.com/logseq/logseq/pull/10426)
	- Fix filesystem path handling, add missing Windows UNC path support [#10525](https://github.com/logseq/logseq/pull/10525)
	- Fix query builder construction [#10485](https://github.com/logseq/logseq/pull/10485)
- [[Enhancement]]
	- Export asset ref link as dataset value for the pdf asset [#10488](https://github.com/logseq/logseq/pull/10488)
	- I18n: Brazilian Portuguese [#10461](https://github.com/logseq/logseq/pull/10461), French [#10454](https://github.com/logseq/logseq/pull/10454), Italian [#10557](https://github.com/logseq/logseq/pull/10557), Turkish [#10492](https://github.com/logseq/logseq/pull/10492)
	- Polish accent color details [#10520](https://github.com/logseq/logseq/pull/10520)

### CODE REVIEW

**Comment by logseqramses** [comment]
I've renamed "The new cmdk" to "The new search menu" as "cmdk" is not descriptive and "cmd" is only for MacOS devices. We don't want to leave out Windows and Linux users, right?

**Comment by logseqramses** [comment]
@andelf @tiensonqin I think we should also include the new theme switcher in the Features section?

**Comment by Bad3r** [comment]
> @andelf @tiensonqin I think we should also include the new theme switcher in the Features section?

That's a great idea. Many users won't know about it till they open the settings. 

Many users might want to switch to it from 3rd party themes; it might be a good idea to consider adding in app popup that showcases the feature.

File: pages/Search.md [hunk 1]
```diff
@@ -2,9 +2,12 @@ type:: [[Feature]]
 platforms:: [[All Platforms]]
 
 - ## Usage
-	- Search pages and full text search blocks by clicking on the search bar at the top or press `Cmd-k`.
-	- To search blocks within a page, run the command `Search in the current page` or press `Cmd-shift-k`.
-	- Pages can also be created from search by typing a new unique string and pressing `Enter`.
+	- Search pages, blocks, files and commands by clicking on the search bar in the top left or press `Cmd-k`.
```
Comment (logseqramses) [inline pages/Search.md]: We should mention what's the shortcut for Windows and Linux is.

**Pull request merged**